decision-records/007-javascript-file-formats.md

```diff
@@ -0,0 +1,84 @@
+# JavaScript file names and formats
+
+## Decision
+
+We will consolidate all published JavaScript in the [`govuk-frontend` npm package](https://www.npmjs.com/package/govuk-frontend) into a single `govuk` directory.
+
+We will publish JavaScript in the following formats:
+
+1. ECMAScript (ES) modules
+2. ECMAScript (ES) modules, bundled
+3. Universal Module Definition (UMD), bundled
+
+We will use ECMAScript (ES) modules by default and only bundle JavaScript files that are considered "entry points" into GOV.UK Frontend, such as `all.mjs` and exported components (for example `accordion.mjs`).
+
+We will deprecate the Universal Module Definition (UMD) format in a future release.
```

Comment:
We talked about this, but I think we should focus decision records on things we've decided to do now, and avoid making future-looking statements.

(I'd be happy to mention it as a future possibility elsewhere in the doc, just not including it in the decision)

Reply:
Ah this was one of @romaricpascal's decisions that we'd deprecate UMD + window globals in future

Will remove